service/src/test/java/com/ge/predix/controller/test/ResourcePrivilegeManagementControllerIT.java

```diff
@@ -100,6 +107,25 @@ public void testResourceInvalidMediaTypeResponseStatusCheck() throws Exception {
     }
 
     @Test
+     public void resourceZoneDoesNotExistException() throws Exception {
+         // NOTE: To throw a ZoneDoesNotExistException, we must ensure that the AcsRequestContext in the
+        //       SpringSecurityZoneResolver class returns a null ZoneEntity
+         MockSecurityContext.mockSecurityContext(null);
+         MockAcsRequestContext.mockAcsRequestContext(this.testZone);
```

Comment:
No need to mock request context again.

Reply:
We need to mock the request context again as this call makes changes to map that is accessed by the SpringSecurityZoneResolver based on the zone name passed to security context. The argument passed is not used inside the code so refactored the mock request context method as well